apps/studio/data/privileges/column-privileges-query.ts

```diff
@@ -18,21 +19,19 @@ export async function getColumnPrivileges(
 ) {
   if (!projectRef) throw new Error('projectRef is required')
 
-  const headers = new Headers()
-  if (connectionString) headers.set('x-connection-encrypted', connectionString)
+  const { sql, zod } = pgMeta.columnPrivileges.list()
 
-  const { data, error } = await get('/platform/pg-meta/{ref}/column-privileges', {
-    params: {
-      path: { ref: projectRef },
-      // this is needed to satisfy the typescript, but it doesn't pass the actual header
-      header: { 'x-connection-encrypted': connectionString! },
+  const { result } = await executeSql(
+    {
+      projectRef,
+      connectionString,
+      sql,
+      queryKey: ['column-privileges', 'list'],
     },
-    signal,
-    headers,
-  })
+    signal
+  )
 
-  if (error) handleError(error)
-  return data
+  return zod.parse(result)
```

Comment:
Changing read queries is out of scope for this task.

But if you want to do it, we probably need additional error handling for failures to parse result.

Reply:
@jgoux I checked that no other read queries are using zod.parse right now. We only use it in tests.

So I've reverted this change. If it becomes a priority in the future, we can make a separate PR with more extensive testing.